lib/src/Uncertainty/Algorithm/Simulation/openturns/NAIS.hxx

```diff
@@ -44,8 +44,8 @@ public:
   NAIS();
 
   /** Default constructor */
-  NAIS(const RandomVector & event,
-       const Scalar rhoQuantile = 0.7);
+explicit  NAIS(const RandomVector & event,
+       const Scalar rhoQuantile = 0.25);
```

Comment:
IMO this should be a key in ResourceMap&openturns.conf.in

Reply:
Okay. We modified openturns.conf.in, ResourceMap.cxx and NAIS.hxx. After compilation, it seems to work for the default value of rhoQuantile. 

In the Resourcemap documentation, we expected a newline with NAIS parameter and the associated value but it's not the case.

We have no idea how to fix it. Any idea?

Thanks !